src/components/items/items.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import './items.css';
+import cheese from './cheese.jpg';
 
+console.log(cheese)
```

Comment:
Yep wow, I forgot to remove the console.log. At least this is a good way to show how to to leave comments.